mscore/scoreview.cpp

```diff
@@ -3613,12 +3613,6 @@ void ScoreView::contextPopup(QContextMenuEvent* ev)
                   objectPopup(gp, e);
                   }
             }
-      else {
```

Comment:
Would this syntax work?
<code>      else {
            QMenu* popup = new QMenu();
            popup->addAction(getAction("append-measure"));
            popup->addAction(getAction("append-measures"));          
            popup->addAction(getAction("append-hbox"));         
            _score->setLayoutAll();     
            _score->update();  
            popup->popup(gp);
            }</code>

Obviously more actions than those three would be wanted, but I don't know if I'm on the right track.